birdhouse/env.local.example

```diff
@@ -189,6 +189,10 @@ export GEOSERVER_ADMIN_PASSWORD="${__DEFAULT__GEOSERVER_ADMIN_PASSWORD}"
 # within the autodeploy docker container as the root user.
 #export AUTODEPLOY_CODE_OWNERSHIP="1000:1000"
 
+# Extra options for 'pavics-compose.sh up -d'.
```

Comment:
This is a nitpick, but it should be `'pavics-compose.sh up'` since `-d` itself is an extra optional flag

Reply:
Agreed !  Too used to always use `up` with `-d` that I just type it without thinking about it.  In fact, I think for my `env.local`, I will bake `-d` in that new var to ensure I never forget to use `-d` !

If you forget to use `-d`, it goes to the foreground, and the only way out is Ctrl-C which will stop the entire stack, so annoying.